config/locales/en.yml

```diff
@@ -406,6 +406,8 @@ en:
     new_spreadsheet: New Board
     no_item_was_found: No item was found.
     no_user_items: No user items.
+    no_user_pages: No page in this board.
```

Comment:
maybe a message more friendly like pt-br one?
"You don't have page in this board yet" or something